cypress/utils/commands.js

```diff
@@ -2,6 +2,21 @@
  * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
  */
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { BASE_PATH } from './constants';
+
+// This function does not delete all indices
```

Comment:
can we modify this now to actually delete all indices and not a specific string?